tools/model-compiler/src/main/java/io/spine/tools/compiler/validation/ValidatingBuilderWriter.java

```diff
@@ -87,6 +88,7 @@ void write(VBType type) {
                                    messageBuilderClassName,
                                    methodsAssembler.createMethods())
                         .addAnnotation(generatedBySpineModelCompiler())
+                        .addAnnotation(canIgnoreReturnValue())
```

Comment:
There is also a static `newBuilder` method, which return value must NOT be ignored.

Therefore your approach is way too broad. Let's discuss in person if needed.